From 7ca077f47552f3d81cbcbd47221b50477c6d3cc9 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Mon, 8 Feb 2021 06:29:04 -0800
Subject: [PATCH 1249/1921] arch/arm64: dts: Reduce the speed of AP eMMC on
 CN9130-CRB

The eMMC card connected to AP SDHCI controller on CN9130-CRB
board does not support HS400 timing. This patch removes the
HS400 DTS property for switching this device to the supported
DDR52 timing.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I0f03dface28c0a415c2274bb192dad7c7cff43b8
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/45554
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dts b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
index 3d50b97fe126..542570b51309 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dts
@@ -84,7 +84,6 @@
 	bus-width = <8>;
 	vqmmc-supply = <&ap0_reg_mmc_vccq>;
 	mmc-ddr-1_8v;
-	mmc-hs400-1_8v;
 	status = "okay";
 };
 
-- 
2.31.1

